.travis.yml

```diff
@@ -16,23 +16,18 @@ env:
 matrix:
   fast_finish: true
   include:
-  - php: 5.6
-    env: MAGENTO_VERSION=2.0.12
-  - php: 5.6
-    env: MAGENTO_VERSION=2.0.*
-  - php: 7.0
-    env: MAGENTO_VERSION=2.1.0
-  - php: 7.0
-    env: MAGENTO_VERSION=2.1.*
   - php: 7.1
     env: MAGENTO_VERSION=2.2.0
   - php: 7.1
-    env: MAGENTO_VERSION=2.2.* CODE_QUALITY=true
+    env: MAGENTO_VERSION=2.2.*
   - php: 7.2
     env: MAGENTO_VERSION=2.3.0
   - php: 7.2
     env: MAGENTO_VERSION=2.3.*
-
+  - php: 7.3
+    env: MAGENTO_VERSION=2.3.0 CODE_QUALITY=true
+  - php: 7.4
```

Comment:
Hmm Magento does not support PHP 7.4 yet, right? Does that still make sense?

Reply:
Thats true this was just a tipo. I have fixed it. But I still have the problem that I cannot run the travis ci test. Hope we can fix this to get the version ready.